modules/memegenBidAdapter.js

```diff
@@ -9,6 +9,12 @@ const openRtbAdapterHost = 'whichtalk.com';
 var timeout;
 var bidsRequested;
 
+String.prototype.removeCharAt = function (i) {
```

Comment:
seems like an overkill to remove the last char

Reply:
it's not the last. i search for the last appearance of ")"

Reply:
which in this case, is the last..